UM/Scene/Camera.py

```diff
@@ -105,7 +103,7 @@ def _updatePerspectiveMatrix(self):
         projection_matrix = Matrix()
         if self.isPerspective():
             if view_width != 0 and view_height != 0:
-                projection_matrix.setPerspective(30, view_width / view_height, 1, 500)
+                projection_matrix.setPerspective(30 + self._zoom_factor, view_width / view_height, 1, 500)
```

Comment:
The zoom of the perspective camera is done by setting the position and not by changing the projection matrix.